addons/crm/models/res_partner.py

```diff
@@ -72,7 +72,7 @@ def _compute_meeting(self):
 
             # Keep only valid meeting data based on record rules of events
             events = [row[1] for row in meeting_data]
-            events = self.env['calendar.event'].search([('id', 'in', events)]).ids
+            events = set(self.env['calendar.event'].search([('id', 'in', events)]).ids)
```

Comment:
What about a distinct in the sql query ?

Reply:
How will a distinct in the query improve the perfs of the meeting_data list comprehension 🤔 ?